frontend/src/components/Header.tsx

```diff
@@ -126,7 +162,7 @@ const NavLinks = ({
           }
           // mark as current if any child page is active
           return children.some((child) => {
-            return child?.href && isCurrentPath(child.href, currentPath);
+            return child?.isCurrentPath(child.href, currentPath);
```

Comment:
this seems like a typo, and throws an error

Reply:
it definitely is haha